ark/utils/segmentation_utils.py

```diff
@@ -239,7 +235,6 @@ def save_segmentation_labels(segmentation_dir, data_dir, output_dir,
             )
 
```

Comment:
Hm I'm a little confused actually what the "cell border image" is supposed to be.

In lines 222-224:
```
# save the cell border image
save_path_seg_borders = os.path.join(output_dir, f'{fov}_segmentation_borders.tiff')
io.imsave(save_path_seg_borders, labels, plugin="tifffile", check_contrast=False)
```

It seems like it's just saying "labels" again? Is "cell border image" supposed to be an image where it is black everywhere and borders are white? Because I don't think that's what's currently being done.